### ISSUE TITLE
Unable to link Postgres with precise environment variable name

### ISSUE THREAD

**Issue by cosmin** [dokku/dokku-postgres#272]
## Description of problem

Unable to link Postgres with custom environment variable name, `_URL` gets appended to the end of the name specified with `-a`. 

I can work around this by modifying the app, but it would be much better if I didn't have to modify the app and instead I could have Dokku link with a precise environment variable name. 

### Steps to Reproduce

1. `dokku postgres:link myappdb myapp -a DATABASE_SERVER`
2. `dokku config:show myapp`

#### Actual Results

```
DATABASE_SERVER_URL:  postgres://postgres:xxxxxxxxxx@dokku-postgres-myappdb:5432/myappdb
```

#### Expected Results

```
DATABASE_SERVER:  postgres://postgres:xxxxxxxxxx@dokku-postgres-myappdb:5432/myappdb
```

**Comment by josegonzalez** [comment]
Yeah this is by design at the moment. The `_URL` always gets appended.
